_data/meltano/utilities/tableau/tableau.yml

```diff
@@ -0,0 +1,52 @@
+commands:
```

Comment:
@janhoon this file should be named after the variant name. Could you rename it to `gabilew.yml` instead?